lib/hammer_cli_foreman/template.rb

```diff
@@ -55,6 +55,15 @@ class InfoCommand < HammerCLIForeman::InfoCommand
         field :locked, _("Locked"), Fields::Boolean
         HammerCLIForeman::References.operating_systems(self)
         HammerCLIForeman::References.taxonomies(self)
+        collection :template_combinations, 'Template Combinations' do
+          field :id, _('ID')
+          field :provisioning_template_id, _('Provisioning template ID')
+          field :provisioning_template_name, _('Provisioning template name')
```

Comment:
`:provisioning_template*` could be excluded as it is always same as the template. Also we usually do not include the other ids here so that the output is not too long.